build/config.props

```diff
@@ -21,7 +21,7 @@
     <!-- Check the VS schedule and manually enter a preview number here that makes sense. -->
     <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">rc</ReleaseLabel>
 
-    <IsEscrowMode>false</IsEscrowMode>
```

Comment:
What is `escrow` here? Usually this term is connected to product/service/money.

Reply:
That's where it's coming from, see the insertion playbook in https://github.com/NuGet/Client.Engineering/issues/new?template=insertion.md and  https://microsoft.sharepoint.com/teams/NuGet/_layouts/OneNote.aspx?id=%2Fteams%2FNuGet%2FTeam%2FNuGet&wd=target%28Engineering.one%7C219AABAA-18B3-46F4-83E8-D3188D83DE22%2FIntegrating%20NuGet%20into%20VS%20through%20VSTS%20%28insertion%5C%29%7C88387971-240E-4ACE-B553-5AEBFF1A7BA6%2F%29
onenote:https://microsoft.sharepoint.com/teams/NuGet/Team/NuGet/Engineering.one#Integrating%20NuGet%20into%20VS%20through%20VSTS%20(insertion)&section-id={219AABAA-18B3-46F4-83E8-D3188D83DE22}&page-id={88387971-240E-4ACE-B553-5AEBFF1A7BA6}&end. 

Look at the issue in https://github.com/NuGet/Client.Engineering/issues/568 and follow the insertion issue for more context.

It's basically implies a higher bar for any changes.